### PR TITLE
refactor tokenissuer

### PR DESCRIPTION
**Pull request by tharindu1st**
refactor tokenissuer

### CODE REVIEW

File: common-controller/internal/operator/controllers/dp/tokenissuer_controller.go [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+	// 		jwtKey.Namespace, jwtKey.Name)
+	// }
+	// jwtIssuerMapping, err := getJWTIssuers(ctx, r.client, jwtKey)
+	// if err != nil {
+	// 	loggers.LoggerAPKOperator.ErrorC(logging.PrintError(logging.Error2660, logging.CRITICAL, "Unable to find associated JWTIssuers: %s", err.Error()))
+	// 	return ctrl.Result{}, err
+	// }
+	// UpdateEnforcerJWTIssuers(jwtIssuerMapping)
+	// return ctrl.Result{}, nil
+}
```
Comment (pubudu538) [inline common-controller/internal/operator/controllers/dp/tokenissuer_controller.go:135]: Let's remove commented code.

File: gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/subscription/ResolvedJWKS.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package org.wso2.apk.enforcer.subscription;
```
Comment (pubudu538) [inline gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/subscription/ResolvedJWKS.java:1]: License header? Please check other locations.

File: gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/discovery/subscription/Event.java [hunk 3]
```diff
@@ -1465,6 +1534,125 @@ public org.wso2.apk.enforcer.discovery.subscription.SubscriptionOrBuilder getSub
...
+    /**
+     * <code>.wso2.discovery.subscription.TokenIssuer tokenIssuer = 8;</code>
+     * @return The tokenIssuer.
+     */
+    public org.wso2.apk.enforcer.discovery.subscription.TokenIssuer getTokenIssuer() {
+      if (tokenIssuerBuilder_ == null) {
+        return tokenIssuer_ == null ? org.wso2.apk.enforcer.discovery.subscription.TokenIssuer.getDefaultInstance() : tokenIssuer_;
+      } else {
+        return tokenIssuerBuilder_.getMessage();
+      }
```
Comment (ShammiL) [inline gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/discovery/subscription/Event.java:1557]:
```suggestion
      if (tokenIssuerBuilder_ == null) {
        return tokenIssuer_ == null ? org.wso2.apk.enforcer.discovery.subscription.TokenIssuer.getDefaultInstance() : tokenIssuer_;
      return tokenIssuerBuilder_.getMessage();
```

Comment (ShammiL) [inline gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/discovery/subscription/Event.java:1557]:
```suggestion
      if (tokenIssuerBuilder_ == null) {
        return tokenIssuer_ == null ? org.wso2.apk.enforcer.discovery.subscription.TokenIssuer.getDefaultInstance() : tokenIssuer_;
      }
      return tokenIssuerBuilder_.getMessage();
```

**Pull request closed without merging**